tools-v2/pkg/cli/command/curvebs/export/target/target.go

```diff
@@ -0,0 +1,129 @@
+/*
+*  Copyright (c) 2023 NetEase Inc.
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+ */
+/*
+* Project: curve
+* Created Date: 2023-12-11
+* Author: wsehjk
+ */
+
+package target
+
+import (
+	"encoding/json"
+	"fmt"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+	"github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/status/client"
+	"github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/status/cluster"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/spf13/cobra"
+)
+
+const (
+	targetexample = "$curve bs export target"
+)
+type ResultStruct map[string]interface{}
+type targetcommand struct {
+	basecmd.FinalCurveCmd
+}
+// check interface targetcommand implement FinalCurveCmdFunc
+var _ basecmd.FinalCurveCmdFunc = (*targetcommand)(nil)  
+
+func ExtractClusterResult(clusterResult map[string]interface{}, service string) ResultStruct{
+	result := ResultStruct{}
+	arrs := clusterResult[service].([]map[string]string)
+	result["label"] = map[string]string{
```

Comment:
Instead of hardcoding, define the constants in `"github.com/opencurve/curve/tools-v2/internal/utils"`.